package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1253,11 +1262,50 @@ class Tempfactors(AtomAttr):
     singular = 'tempfactor'
     per_object = 'atom'
     dtype = float
+    transplants = defaultdict(list)
+    aliases = [{"attrname": "bfactors", "singular": "bfactor"}]
 
     @staticmethod
     def _gen_initial_values(na, nr, ns):
         return np.zeros(na)
 
+    def bfactor(self):
+        """Tempfactor alias property for atom
+
+        .. versionadded:: 2.0.0
+        """
+        return self.universe.atoms[self.ix].tempfactor
+
+    def bfactor_setter(self, value):
+        """Tempfactor alias property for atom
+
+        .. versionadded:: 2.0.0
+        """
+        self.universe.atoms[self.ix].tempfactor = value
+
+    def bfactors(self):
+        """Tempfactor alias property for groups of atoms
+
+        .. versionadded:: 2.0.0
+        """
+        return self.universe.atoms[self.atoms.ix].tempfactors
+
+    def bfactors_setter(self, value):
+        """Tempfactor alias property for atom
+
+        .. versionadded:: 2.0.0
+        """
+        self.universe.atoms[self.atoms.ix].tempfactors = value
+
+    transplants[Atom].append(
+        ('bfactor', property(bfactor, bfactor_setter, None,
+                             bfactor.__doc__)))
+    
+    for group in (AtomGroup, Residue, ResidueGroup, Segment, SegmentGroup):
+        transplants[group].append("bfactors",
+                                  property(bfactors, bfactors_setter, None,
+                                           bfactors.__doc__))
```

Comment:
```suggestion
        transplants[group].append(
            ("bfactors", property(bfactors, bfactors_setter, None,
                                  bfactors.__doc__)))
```
? I _think_ this is why CI is breaking?

Reply:
Whoooops thanks